src/components/RentalsOverview/RentalsOverview.tsx

```diff
@@ -0,0 +1,58 @@
+import RentalOverviewItem from 'components/RentalsOverviewItem/RentalsOverviewItem';
+import RentalModel from 'models/RentalModel';
+import React, { useEffect, useState } from 'react';
+import classes from './RentalsOverview.module.scss';
+import rentalsService from 'services/rentalsService';
+import tokenService from 'services/tokenService';
+import { useLocation } from 'react-router-dom';
+
+interface RentalsOverviewProps {
+  componentType: 'current' | 'history';
+}
+
+const RentalsOverview = ({ componentType }: RentalsOverviewProps) => {
+  const [rentals, setRentals] = useState<RentalModel[]>([]);
+  const { getRentals, getRentalsByUser } = rentalsService;
+  const { isRoleAdmin } = tokenService;
+  const isAdmin = isRoleAdmin();
+
+  const location = useLocation();
+  const userId: number = parseInt(location.pathname.split('/')[2]);
```

Comment:
There is no need to parse url to get query param, as I mentioned in previous comment there is hook that can do that